enterprise/admin/views.py

```diff
@@ -250,6 +250,11 @@ def _enroll_users(cls, enterprise_customer, emails, course_id, mode, request):
                     dict(user=username, message=error_message),
                 )
             else:
+                ecu = EnterpriseCustomerUser.objects.get_link_by_email(email)
```

Comment:
No; we shouldn't get here unless we have a fully linked `EnterpriseCustomerUser`, in which case we want to fail loudly.